src/HttpClientInterception/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,3 @@
 #nullable enable
+JustEat.HttpClientInterception.HttpRequestInterceptionBuilder.For(params System.Func<System.Net.Http.HttpRequestMessage!, System.Threading.Tasks.Task<bool>!>![]! predicates) -> JustEat.HttpClientInterception.HttpRequestInterceptionBuilder!
+JustEat.HttpClientInterception.HttpRequestInterceptionBuilder.For(params System.Predicate<System.Net.Http.HttpRequestMessage!>![]! predicates) -> JustEat.HttpClientInterception.HttpRequestInterceptionBuilder!
```

Comment:
Once the PR is otherwise approved, these should move into the `.Shipped.txt` file.

Reply:
I've moved these APIs to `.shipped.txt`. Please let me know if there's anything else to improve this PR.